go.mod

```diff
@@ -12,16 +12,16 @@ require (
 	golang.org/x/mod v0.5.1
 	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 	gotest.tools/v3 v3.0.3
-	k8s.io/api v0.22.5
-	k8s.io/apiextensions-apiserver v0.22.5
-	k8s.io/apimachinery v0.22.5
+	k8s.io/api v0.23.4
+	k8s.io/apiextensions-apiserver v0.23.4
+	k8s.io/apimachinery v0.23.4
 	k8s.io/cli-runtime v0.22.5
```

Comment:
The failed build should be fixed by upgrading `cli-runtime` to matching k8s version.

Reply:
https://github.com/knative/client/pull/1632/commits/14d772350172ff6eea1aab1f3df870dfad0a7900